Products/Archetypes/skins/archetypes/edit_macros.pt

```diff
@@ -80,11 +80,12 @@
             method="post"
             enctype="multipart/form-data"
             class="enableUnloadProtection enableAutoFocus atBaseEditForm"
+            data-pat-autotoc="levels: legend; section: fieldset; className: autotabs"
             action=""
             id=""
             tal:attributes="action python:context.absolute_url()+'/'+template.id;
                             id string:${portal_type}-base-edit;
-                            class python:test(path('allow_tabbing|nothing'), 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection', 'enableUnloadProtection enableAutoFocus enableUnlockProtection');">
+                            class python:test(path('allow_tabbing|nothing'), 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection pat-autotoc', 'enableUnloadProtection enableAutoFocus enableUnlockProtection');">
```

Comment:
Great lengths have been ran recently because of such "sugar" (e.g. applying ZCA decorators in all pkgs) but I agree this should be done in all templates, in a separate PR. This one lgtm.